src/Test/StrongCheck/Laws/Data/DivisionRing.purs

```diff
@@ -0,0 +1,37 @@
+module Test.StrongCheck.Laws.Data.DivisionRing where
+
+import Prelude
+
+import Effect (Effect)
+import Effect.Console (log)
+import Test.StrongCheck (quickCheck')
+import Test.StrongCheck.Arbitrary (class Arbitrary)
+import Type.Proxy (Proxy)
+
+-- | Non-zero ring: one /= zero
+-- | Non-zero multiplicative inverse: recip a * a = a * recip a = one for all non-zero a
+checkDivisionRing
+  ∷ ∀ a
+  . DivisionRing a
+  ⇒ Arbitrary a
+  ⇒ Eq a
+  ⇒ Proxy a
+  → Effect Unit
+checkDivisionRing _ = do
+
+  log "Checking 'Non-zero ring' law for DivisionRing"
+  quickCheck' 1000 nonZero
+
+  log "Checking 'Non-zero multiplicative inverse' law for DivisionRing"
+  quickCheck' 1000 inverse
+
+  where
+
+  nonZero ∷ Boolean
+  nonZero = (one :: Eq a => a) /= zero
+
+  inverse ∷ DivisionRing a => a → Boolean
+  inverse a =
```

Comment:
We perhaps need to add a `if a == zero then true` guarding case to this? Since the law only applies for non-zero `a`.

Reply:
I suppose that's true -- here's the relevant excerpt from Pursuit:

> The DivisionRing class is for non-zero rings in which every non-zero element has a multiplicative inverse. Division rings are sometimes also called skew fields.
> 
> Instances must satisfy the following laws in addition to the Ring laws:
> 
>     Non-zero ring: one /= zero
>     Non-zero multiplicative inverse: recip a * a = a * recip a = one for all non-zero a
> 
> The result of recip zero is left undefined; individual instances may choose how to handle this case.
> 
> If a type has both DivisionRing and CommutativeRing instances, then it is a field and should have a Field instance.

I have both checks, but the inverse doesn't verify it's not zero. That said, all the tests passed, and I'm assuming they _must_ have included zero substituted in, so that's interesting. I can make a quick update.

Reply:
I guess even if they did pass with `0.0` it might not necessarily be true for other types, since the behaviour is said to be undefined.

Could you raise the minimum strongcheck dependency to 4.1.0 while you're at it too? Thanks!

Reply:
👍 Done!